### PR TITLE
client:get metadata from manager

### PR DESCRIPTION
**Pull request by HHoflittlefish777**
#17 
This is a preliminary function, and subscription needs to be implemented later

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -25,6 +25,7 @@ dashmap = "5.4.0"
 client = {path = "client"}
 server = {path = "server"}
 manager = {path = "manager"}
+common = {path = "common"}
```
Comment (luanshaotong) [inline Cargo.toml:28]: What is the point of this?

**Comment by luanshaotong** [comment]
You can solve the conflicts or I'll directly merge this into my develop branch.

**Comment by HHoflittlefish777** [comment]
@luanshaotong confict has been solved

**Comment by HHoflittlefish777** [comment]
@luanshaotong please merge it

**Review by luanshaotong — APPROVED** [review]
LGTM, and we should wrap the whole client by tokio then.

**Pull request merged**